common/src/main/java/com/amazonaws/services/schemaregistry/common/AWSSchemaRegistryClient.java

```diff
@@ -114,6 +96,34 @@ public AWSSchemaRegistryClient(@NonNull AwsCredentialsProvider credentialsProvid
             }
         }
         this.client = glueClientBuilder.build();
+
+        //When AWSGlueCrossRegionSchemaReplicationConverter is used
+        //source.registry.name, source.region and source.endpoint are required.
+        //So we need to create another GlueClient with source registry details to retrieve
+        //all schema versions from the source registry schema
+        GlueClientBuilder glueSourceClientBuilder = glueClientBuilder;
+        if (glueSchemaRegistryConfiguration.getSourceRegistryName() != null &&
+                glueSchemaRegistryConfiguration.getSourceRegion() != null &&
+                glueSchemaRegistryConfiguration.getSourceEndPoint() != null) {
+
+            glueSourceClientBuilder = GlueClient
```

Comment:
Instead of creating multiple clients here, can we pass in a region and create multiple instances of this client in different regions?

Reply:
With the new code, no change is required in this section.